app/logic/events.py

```diff
@@ -105,7 +152,7 @@ def attemptSaveEvent(eventData, attachmentFiles = None, renewedEvent = False):
     If it is not valid it will return a validation error.
 
     Returns:
-    Created events and an error message.
+    The saved event Created events and an error message if an error occurred.
```

Comment:
Nit: Reword to "The saved event, created events, and an error message if an error occured."